src/rulesets/Base/base.jl

```diff
@@ -88,3 +88,55 @@ function rrule(::typeof(identity), x::Tuple)
     end
     return x, identity_pullback
 end
+
+#####
+##### `evalpoly`
+#####
+
+if VERSION ≥ v"1.4"
+    function frule((_, Δx, Δp), ::typeof(evalpoly), x, p)
+        q = _evalpoly_dxcoef(p)
+        return evalpoly(x, p), evalpoly(x, Δp) + evalpoly(x, q) * Δx
+    end
+
+    function rrule(::typeof(evalpoly), x, p)
+        function evalpoly_pullback(Δy)
+            ∂x = Thunk() do
+                q = _evalpoly_dxcoef(p)
+                return evalpoly(x, q)' * Δy
+            end
+            ∂p = @thunk _evalpoly_backp(Δy, x, p)
+            return NO_FIELDS, ∂x, ∂p
+        end
+        return evalpoly(x, p), evalpoly_pullback
+    end
+
+    _evalpoly_dxcoef(p) = ntuple(i -> i * p[i + 1], length(p) - 1)
+    function _evalpoly_dxcoef(p::AbstractVector)
+        N = length(p)
+        @inbounds q = (1:(N - 1)) .* view(p, 1:(N - 1))
+        return q
+    end
+
+    # This is a geometric progression, that is ∂p = Δy * (I, x, x², …, xⁿ)'
+    # TODO: Handle case where x is a matrix and p is UniformScaling
+    function _evalpoly_backp(Δy, x, p::Tuple)
+        N = length(p)
+        ∂p = ntuple(_ -> Δy, N)
+        x′ = x'
```

Comment:
Should this really be adjoint and not transpose?

```suggestion
        x′ = transpose(x)
```

Reply:
Assuming ChainRules is following the same conventions with complex numbers as Zygote does, then yes this should be.

Reply:
ChainRules does not follow the same convention a Zygote.
Its a thing we could change, but right now we work with the gradient of complex numbers, not its conjugate.

I don't actually know if we have an issue for that, feel free to open one on ChainRulesCore if we don't.